index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body class="all">
+    <header>
+        <nav>
+            <img class="navbar" src="./images/Bars.svg" alt="bars"> 
+        </nav>
+    </header>
+    <section class="section-spacing">
+        <div class="introduction">
+            <div class="texts">
+                <h1 class="welcome">Hey there.<br/> I'm Assem.<br/>
+                  <span class="green">I'm a software developer</span></h1>
+                <p class="profession"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                  don't hesitate to contact me.</p>
+            </div>
+            <div class="socialmedia">
+                <ul>
+                    <li><a href="#"><img src="./images/twitter.svg" alt="Twitter"></a></li>
+                    <li><a href="#"><img src="./images/Linkedin icon.svg" alt="Linkedin"></a></li>
+                    <li><a href="#"><img src="./images/medium.svg" alt="Medium"></a></li>
+                    <li><a href="#"><img src="./images/github.svg" alt="Github"></a></li>
+                    <li><a href="#"><img src="./images/angellist.svg" alt="Enabled"></a></li>
+                </ul>
+            </div>
+            <div class="scroll">
+                <img src="./images/Disabled.png" alt="scroll button">
+            </div>
+        </div>
+    </section>
+    <section class="works">
+        <div class="recent-works">
+           <h2>My Recent Works</h2>
+           <img class="indicator" src="./images/Indicator.svg" alt="Indicator">
+        </div>
+        <div class="individual-card-item-flex">
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
```

Comment:
- [ ] Well done on implementing the `<li>` elements, but there are some improvements needed to align them with the design. The font styles, including font size, font weight, letter spacing, color, and font family, should be adjusted to match the design specifications. Additionally, the background color and border should be modified as per the design. Instead of using a border, achieve the desired effect by applying the appropriate background color and padding to the elements. By making these adjustments, you will ensure that the `<li>` elements visually align with the design and create a more cohesive and consistent look. Keep up the great work! 👍

_**Note: This request applies for all languages "li" elements in the project section. 👍**_